plugins/woocommerce/includes/class-wc-checkout.php

```diff
@@ -451,6 +454,10 @@ public function create_order( $data ) {
 			$order->set_payment_method( isset( $available_gateways[ $data['payment_method'] ] ) ? $available_gateways[ $data['payment_method'] ] : $data['payment_method'] );
 			$this->set_data_from_cart( $order );
 
+			if ( $this->cogs_is_enabled() ) {
```

Comment:
Curious if we should supply `__METHOD__` here?

Reply:
No, because we are inside the normal checkout flow. We only want doing-it-wrong's triggered when a public method that doesn't make sense with the feature disabled is explicitly invoked.